application/web/web.js

```diff
@@ -221,11 +240,25 @@ function saveCurrentEntry() {
     const newEntry = document.createElement('button');
     newEntry.innerText = title;
     const article = document.createElement('article');
-    article.setAttribute('id', new Date());
+
+    const date = new Date();
+    let day = date.getDate();
+    let month = date.getMonth() + 1;
+    let year = date.getFullYear();
+
+    newEntry.setAttribute('id', `${year}-${month}-${day}-title`);
+    article.setAttribute('id', `${year}-${month}-${day}`);
     article.innerText = entry;
     article.style.display = 'none';
+
+    // TODO: Replace with function to load from storage
     newEntry.addEventListener('click', () => {
-        const article = document.querySelector(`#${new Date}`);
+        const date = new Date();
+        let day = date.getDate();
+        let month = date.getMonth() + 1;
+        let year = date.getFullYear();
+
+        const article = document.querySelector(`[id='${year}-${month}-${day}']`);
```

Comment:
Just realized that the saving functionality needs to be implemented for the third bullet point to work. Could you comment or remove this code, so that Nick can just focus on it.